src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml

```diff
@@ -0,0 +1,101 @@
+<documentation title="Array Indent">
+    <standard>
+    <![CDATA[
+    The opening square bracket/parenthesis of a multi-line array must be indented at least to the same level of the start of the statement.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Opening square bracket/parenthesis token of a multi-line array indented at least to the same level of the start of the statement.">
+        <![CDATA[
+if ($condition) {
+    $a =
+<em>    [</em>
+        1,
+        2,
+    ];
+}
+        ]]>
+        </code>
+        <code title="Invalid: Opening square bracket/parenthesis of a multi-line array not indented at least to the same level of the start of the statement.">
+        <![CDATA[
+if ($condition) {
```

Comment:
Initially, I wanted to use the snippet below as the invalid example but the generator code trims the leading spaces before `$a` (https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/8974f2662ac60cc2d6a545eda35e41c6f0fb62e8/src/Generators/Text.php#L131) and that is why I opted to add an `if` statement.

```php
    $a =
[
        1,
        2,
    ];
```